docs/source/usage.rst

```diff
@@ -0,0 +1,253 @@
+=====
+Usage
+=====
+
+This toolkit contains the tools to interface with HOLE_
+:cite:p:`Smart1993,Smart1996` to analyse an ion channel pore or transporter
+pathway :cite:p:`Stelzl2014`.
+
+Using HOLE on a PDB file
+------------------------
+
+Use the ``hole`` function to run `HOLE`_ on a single PDB file. For example,
+the code below runs the `HOLE`_ program installed at `~/hole2/exe/hole`::
+
+    from MDAnalysis.tests.datafiles import PDB_HOLE
+    from MDAnalysis.analysis import hole2
+    profiles = hole2.hole(PDB_HOLE, executable='~/hole2/exe/hole')
+    # to create a VMD surface of the pore
+    hole2.create_vmd_surface(filename='hole.vmd')
+
+``profiles`` is a dictionary of HOLE profiles, indexed by the frame number. If
+only a PDB file is passed to the function, there will only be one profile at
+frame 0. You can visualise the pore by loading your PDB file into VMD, and in
+:menuselection:`Extensions --> Tk Console`, type::
+
+    source hole.vmd
+
+You can also pass a DCD trajectory with the same atoms in the same order as
+your PDB file with the ``dcd`` keyword argument. In that case, ``profiles``
+will contain multiple HOLE profiles, indexed by frame.
+
+The HOLE program will create some output files:
+
+    * an output file (default name: hole.out)
+    * an sphpdb file (default name: hole.sph)
+    * a file of van der Waals' radii
+      (if not specified with ``vdwradii_file``. Default name: simple2.rad)
+    * a symlink of your PDB or DCD files (if the original name is too long)
+    * the input text (if you specify ``infile``)
+
+By default (`keep_files=True`), these files are kept. If you would like to
+delete the files after the function is wrong, set `keep_files=False`. Keep in
+mind that if you delete the sphpdb file, you cannot then create a VMD surface.
+
+
+Using HOLE on a trajectory
+--------------------------
+
+You can also run HOLE on a trajectory through the :class:`HoleAnalysis`
+class. This behaves similarly to the ``hole`` function, although
+arguments such as ``cpoint`` and ``cvect`` become runtime arguments for
+the :meth:`~HoleAnalysis.run` function.
+
+The class can be set-up and run like a normal MDAnalysis analysis class::
+
+    import MDAnalysis as mda
+    from MDAnalysis.tests.datafiles import MULTIPDB_HOLE
+    from MDAnalysis.analysis import hole2
+
+    u = mda.Universe(MULTIPDB_HOLE)
+
+    ha = hole2.HoleAnalysis(u, executable='~/hole2/exe/hole') as h2:
```

Comment:
Here and above/below, our recommended approach here is to install hole2 via conda-forge. There doesn't seem to be any instructions for users on how to deal with the CF installation case (i.e. they may not easily know the path to their installation)

Reply:
Thanks for that note Irfan, I've added a section on finding the path. I'll update installation notes when there's a full c-f package.

Reply:
Actually, doesn't hurt to update now, especially if the order of events has to go release -> package...

Reply:
Added some very brief installation notes as well!